modules/sdk-core/src/bitgo/tss/ecdsa/ecdsa.ts

```diff
@@ -121,17 +130,16 @@ export async function createUserSignShare(xShare: XShareWithNtilde, yShare: YSha
  * Creates the Gamma Share and MuShare with User WShare and AShare From BitGo
  * @param {WShare} wShare User WShare
  * @param {AShare} aShare AShare from Bitgo
- * @returns {Promise<CreateUserGammaAndMuShareRT>}
+ * @returns {Promise<SignConvertStep2Response>}
  */
-export async function createUserGammaAndMuShare(wShare: WShare, aShare: AShare): Promise<CreateUserGammaAndMuShareRT> {
+export async function createUserGammaAndMuShare(wShare: WShare, aShare: AShare): Promise<SignConvertStep2Response> {
```

Comment:
TODO: we can remove this fn in the future and add this validation to `signConvertStep2`